app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,76 @@
+class Admin::ReportsController < Admin::ApplicationController
+  before_action :authenticate_admin!
+  before_action :set_report, only: [:show, :edit, :update, :destroy]
+
+  layout 'admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.